mina.netty/src/main/java/org/kaazing/mina/core/service/AbstractIoAcceptor.java

```diff
@@ -106,7 +105,8 @@ public SocketAddress getLocalAddress() {
     @Override
     public final Set<SocketAddress> getLocalAddresses() {
         Set<SocketAddress> localAddresses = new HashSet<>();
-        synchronized (bindLock) {
```

Comment:
I don't think bindLock is required since we always synchronize the same code block on boundAddresses as well. Is it still present in the mina version?